src/Components/Server/src/CircuitOptions.cs

```diff
@@ -46,5 +46,14 @@ public class CircuitOptions
         /// Defaults to <c>3 minutes</c>.
         /// </value>
         public TimeSpan DisconnectedCircuitRetentionPeriod { get; set; } = TimeSpan.FromMinutes(3);
+
+        /// <summary>
+        /// Gets or sets a value that determines whether or not to send detailed exception messages from .NET interop method invocation
+        /// exceptions to JavaScript.
+        /// </summary>
+        /// <remarks>
+        /// The default value for this property is <c>false</c>. This value should only be turned on in development scenarios as turning it
+        /// on in production might result in a sensitive information leak.</remarks>
+        public bool DotNetInteropExceptionDetails { get; set; }
```

Comment:
I think the name we use elsewhere for this is `DetailedErrors` (e.g., there's a `bool` flag with that name in `WebHostOptions`). Can we use the same name? Maybe `JSInteropDetailedErrors` if we want to clarify it's only applicable to JS interop.

Reply:
How do you feel about JSInterop vs DotNetInterop, do you want to put everything under the JSInterop umbrella?

I'm happy calling this JSInteropDetailedErrors, but it really is DotNetInteropDetailedErrors as its going from the JS side to the dotnet side.

Reply:
I'm starting to wonder if this should be configurable at all or if we should just key this out of `WebHostDefaults.DetailedErrorsKey` like webhost does and be done with it.

Reply:
> How do you feel about JSInterop vs DotNetInterop, do you want to put everything under the JSInterop umbrella?

I don't relate to the distinction between "JS interop" and ".NET interop" in the way I think you do. For me, neither naming specifically clarifies which direction the calls are going in, just which technologies are involved. So I'm quite happy just to call this "JS interop" (where it's obvious that the other side of the wire is .NET, since this is .NET code).

Reply:
Ok, lets put everything under the JSInterop umbrella (I already did the changes to reflect that)